examples/all-options.xml

```diff
@@ -196,6 +197,7 @@
           <overlappingCollectives>False</overlappingCollectives> <!-- true if overlapping collectives should be used, false otherwise. REQUIRES MPI>=3! -->
           <overlappingStartAtStep>5</overlappingStartAtStep> <!-- Start overlapping at given step (default: 5), only relevant if overlappingCollectives==True -->
           <overlappingP2P>False</overlappingP2P> <!-- Defines whether to use overlapping p2p communication or not. Default: False -->
+          <boundaries> <x>reflective</x><y>outflow</y><z>reflective</z> </boundaries> <!-- Select boundary type for each dimensions. Available options are reflecting/reflective, outflow and periodic (default)-->
```

Comment:
Any particle interacting with an outflow boundary will get removed, and that will take precedence. Should the behaviour be different?

Basically for each particle, each dimension is checked in order, to determine what happens. So in this case, first the x component of the velocity would be reversed, and then y will be checked, outflow will be found, and the particle would be deleted.

I can add a comment saying that this will happen.

Reply:
When a particle moves exactly through the corner it is debatable if it interacts with the outflowing or the reflective boundary. I don't really care.

One could argue that if the particle's trajectory would go through the reflective boundary but it ends up in the cell behind the corner it should be reflected. Nevertheless, whatever the behavior is, document it precisely.